src/main/scala/scalajs_java/compiler/Compiler.scala

```diff
@@ -35,6 +29,8 @@ class Compiler(val errorHanlder: ErrorHandler) {
   val opCompiler = new OpCompiler(errorHanlder)
 
   val typeCompiler = new TypeCompiler(errorHanlder)
+
+  val compiledInits = inits.mapValues(compileExpr)
```

Comment:
This seems to be unused.

Btw, `mapValues` is tricky. It evaluates the mapped values "lazily", every time you access an element, rather than once for all when `mapValues` is called. In general, stay away from `mapValues` unless you know what you're doing.